semantic.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+ <meta charset="UTF-8">
+ <meta http-equiv="X-UA-Compatible" content="IE=edge">
+ <meta name="viewport" content="width=device-width, initial-scale=1.0">
+ <title>Div soup page</title>
+</head>
+
+<body>
+ <!-- ovo je semanticki napisan kod koji ce jos sadrazavat i komentare  -->
+ <!-- ovo je Header dio koji sadrzi i  logo, naziv i navigaciju  -->
+ <header>
+  <img src="html logo.png" alt="html logo" width="150px">
```

Comment:
@borissimic Bilo bi dobro da je naziv logo slike iz jedne riječi, bez razmaka. Također, `width="150"` je isto kao i ovo što si napisao, jer HTML očekuje da se piše u pikselima, tako da nema potrebe definirati jedinicu veličine.

Reply:
da to je bio propust s nazivom slike .. popravljeno :D